src/uucore/src/lib/features/splice.rs

```diff
@@ -0,0 +1,84 @@
+// This file is part of the uutils coreutils package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+//! This module provides the [`write_fast_using_splice`] function to leverage the `splice` system call
+//! in Linux systems, thus increasing the I/O performance of copying between two file descriptors.
+
+use nix::unistd;
+use std::{
+    io::Read,
+    os::{
+        fd::AsFd,
+        unix::io::{AsRawFd, RawFd},
+    },
+};
+
+#[cfg(any(target_os = "linux", target_os = "android"))]
+use crate::pipes::{pipe, splice, splice_exact};
+
+const SPLICE_SIZE: usize = 1024 * 128;
```

Comment:
please document these "magic numbers"

Reply:
damn not the author of this :smile: hmm

Reply:
sorry but where this code is from ? you can't just take random code from the web and inject it here with our license.

Reply:
yeah. it's taken from there

Reply:
ok, leave it like this then :)

I will have a look

Reply:
sure sure